Matveibos/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+
+# this class is Hash
+class MyHash
+  include Enumerable
+
+  attr_reader :pair_count, :size, :arr
+
+  LOAD_FACTOR = 0.5
+  SIZE = 10
+
+  def initialize
+    initialize_hash
+  end
+
+  def [](key)
+    ident = key.hash % @size
+    arr[ident].find { |k, _| k == key }&.last
```

Comment:
`arr[ident].find { |k, _| k == key }` is also used 2 times -> move it to the method